netkan/netkan/webhooks/github_inflate.py

```diff
@@ -80,11 +81,16 @@ def frozen_ids_from_commits(commits: List[Dict[str, Any]]) -> List[str]:
 
 
 def freeze(ids: List[str]) -> None:
-    with ModStatus.batch_write() as batch:
+    if ids:
         logging.info('Marking frozen mods...')
-        for mod in ModStatus.scan(rate_limit=5):
-            if not mod.frozen and mod.ModIdentifier in ids:
-                logging.info('Marking frozen: %s', mod.ModIdentifier)
-                mod.frozen = True
-                batch.save(mod)
+        for ident in ids:
+            try:
+                status = ModStatus.get(ident)
+                if not status.frozen:
+                    logging.info('Marking frozen: %s', ident)
+                    # https://readthedocs.org/projects/pynamodb/downloads/pdf/stable/
+                    status.update(actions=[ModStatus.frozen.set(True)])
+            except ModStatus.DoesNotExist:
+                # No status, don't need to freeze
+                pass
```

Comment:
This looks good to me. If we ever try to freeze a few 100 mods at once this might fail, but that likely isn't something we need to worry about.